doc/99_glossary.md

```diff
@@ -0,0 +1,23 @@
+# Glossary {#glossary} #
+This glossary gives brief descriptions of some of the key terms used in this software.
+
+### Bluetooth Low Energy (BLE) {#ble} ###
+Bluetooth Low Energy is a specialization of the Bluetooth specification that is designed to be used with devices that need to conserve power.  Since AsteroidOS is designed to run on watches that run on batteries, using BLE is a natural fit.  See https://en.wikipedia.org/wiki/Bluetooth_Low_Energy for more details.
+
+### characteristic {#characteristic} ###
+a single data value in BLE that is transferred between devices.  An example is the current volume setting of media player software.
+
+### connected {#connected} ###
+devices which are paired and both ready to send or receive data are connected.  Note that it is possible for devices to be paired but not connected, but they cannot be connected and unpaired.
+
+### paired {#paired} ###
+two Bluetooth devices which have already completed pairing are said to be paired.
+
+### pairing {#pairing} ###
+the process by which a computer and a watch are "introduced" to each other so that they can begin to communicate via Bluetooth 
```

Comment:
Misses a **.**

Reply:
I've tried to make all of the entries more consistent.  Now they are all proper sentences.